code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -276,3 +276,13 @@
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
 							)
+
+/obj/item/clothing/suit/armor/ego_gear/impending_day
+	name = "impending day"
+	desc = "Even still, I witnessed man and sky and earth tear into thousands of pieces."
+	icon_state = "impending_day"
+	flags_inv = NONE
+	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = -20, BLACK_DAMAGE = 50, PALE_DAMAGE = 20) // 70
```

Comment:
Why not 0 white and 0 pale? Egor is going to have a fit.

Reply:
This is fine, 20 pale is on He, it's the average.